package.json

```diff
@@ -68,12 +68,13 @@
     "lodash": "~4.15.0",
     "multiselect": "~0.9.12",
     "patternfly": "^3.29.5",
-    "patternfly-react": "^0.8.1",
+    "patternfly-react": "^0.11.0",
```

Comment:
all package upgrades seems not related?

Reply:
I've upgraded in order to use Icon component